src/openscm_units/_unit_registry.py

```diff
@@ -354,9 +356,12 @@ def add_standards(self):
             "Tt = 1000000000000 * t"
         )  # since Tt is used for "tex" in the defaults
 
-        self.define("ppt = [concentrations]")
-        self.define("ppb = 1000 * ppt")
-        self.define("ppm = 1000 * ppb")
+        self.define("ppm = [concentrations]")
+        self.define("ppb = ppm / 1000")
+        self.define("ppt = ppb / 1000")
+        # Have to rebuild cache to get right units for ppm as it is defined in
+        # pint
+        self._build_cache()
```

Comment:
Does this still work for older versions of pint? ie. it might be worth adding a test to cover the lowest supported version of pint since there is a bit of flux

Reply:
How far back do we want to go do you reckon?

Reply:
Following loosely [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy.html), which has a window of 24 months for numpy versions, we could support anything which is at most 2 years old, i.e. currently pint >= 0.18.

Reply:
Dunno. Probably not worth putting the effort in to maintain a really old version. pyam pins >=0.13.0. Maybe try that and then bisect if there are any issues

Reply:
> [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy.html),

Ahh nice. Probably best to follow Mika's suggestion, given that it's a standard and easier to defend😄

Reply:
Ok I've pushed this out into an issue because it exploded on me #42. I'm going to revert here because it's non-trivial